mods/gen4/items.js

```diff
@@ -48,8 +48,7 @@ let BattleItems = {
 					priority: action.priority + 0.1,
 					pokemon: action.pokemon,
 					move: action.move,
-					// @ts-ignore
-					target: action.target,
+					target: action.targetLoc,
```

Comment:
This looks like a mechanics bugfix... should probably not be bundled into a bunch of unrelated refactors.

Reply:
In hindsight you might as well, but the correct fix is `targetLoc: action.targetLoc`.